src/H5back_gen.py

```diff
@@ -0,0 +1,472 @@
+import json
```

Comment:
Also since this is meant to be a script, the first line is usually `#!/usr/bin/env python`.  This is called as shebang line, look up what it does.